Testscripts/Windows/CPU-Offline-Online.ps1

```diff
@@ -0,0 +1,154 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+<#
+.Synopsis
+	CPU offline-online stress testing with vmbus interrupt channel reassignment via VM reboot
```

Comment:
what is the difference between this file and CPU-Offline.ps1? Could we use the single PS file?

Reply:
Fixed in ccf8f7d41f4469dbb2b6d03be0ae8d8be1979947